scm/driver/github/repo.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/drone/go-scm/scm"
 )
 
-type repository struct {
+type Repository struct {
```

Comment:
From what I understand, it makes the struct public. Why do we need to make it public if all references are from same package/file?
Sorry, I may be naive here, just trying to understand its impact

Reply:
json unmarshal and marshal only works when name of struct start with capital letter

Reply:
Oh interesting. Thanks for the info.

Reply:
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/10402706/186148207-8c4bec34-c32a-4935-9c4b-7e0582f2faf8.png">
https://github.com/drone/go-scm/pull/219
repositories should be lower case as @mohitg0795 said

Reply:
but then, RepositoryListResponse will not be able to map to Repository struct as it does not unmarshal that struct

Reply:
Can you suggest any other method, so that it will map correctly

Reply:
can you show me where it doesnt work, the unit tests passed and it unmarshalled successfully

if you look at the do method for all drivers, it marshalls / unmarshalls without all of the structs. The other structs all start with a lowercase letter.

Reply:
ya, I have changed them now